lib/components/helpers/required-label.js

```diff
@@ -0,0 +1,38 @@
+// # required label component
+
+/*
+  Required Label for a field
+*/
+
+'use strict';
+
+import React from 'react';
+import createReactClass from 'create-react-class';
+import cx from 'classnames';
+
+import HelperMixin from '../../mixins/helper';
+
+export default createReactClass({
+
+  displayName: 'RequiredLabel',
+
+  mixins: [HelperMixin],
+
+  render: function () {
+    return this.renderWithConfig();
+  },
+
+  renderDefault: function () {
+    var config = this.props.config;
```

Comment:
I know Formatic is a mix of styles, but you can feel free to use `const` and `let` if you want. We need to do a big cleanup pass, so either way is fine.

Reply:
Yeah wasn't sure. When in Rome? I'll just leave this for now and it can be refactored later with all the rest.